developers/advanced/null-value-support.md

```diff
@@ -37,7 +40,35 @@ Remember that in the JSON used as table configuration, `defaultNullValue` must a
 If the column type is not String, Pinot will convert that value to the column type automatically.
 {% endhint %}
 
-## Store nulls at ingestion time
+### Enable basic null support for the single-stage query engine
+
+To enable basic null support (`IS NULL` and `IS NOT NULL`) and generate the null index, in the Table index configuration ([tableIndexConfig](https://docs.pinot.apache.org/configuration-reference/table#tableindexconfig-1)), set `nullHandlingEnabled=true`.
+
+{% hint style="info" %}
+**Important**
+
+You MUST `SET enableColumnBasedNullHandling=true;` before you query. Just having `"nullHandlingEnabled: true,"` set in your table config does not automatically provide `enableColumnBasedNullHandling=true` when you execute a query. Basic null handling supports `IS NOT NULL` and `IS NULL` predicates. Advanced null handling adds SQL compatibility.
+{% endhint %}
+
+### Enable basic null support for the multi-stage query engine
+
+To enable basic null support (`IS NULL` and `IS NOT NULL`) and generate the null index, in the Table index configuration ([tableIndexConfig](https://docs.pinot.apache.org/configuration-reference/table#tableindexconfig-1)), set `enableColumnBasedNullHandling=true`.
```

Comment:
This is incorrect and looks just copied and pasted from single-stage query engine